swagger_to_sdk_config.json

```diff
@@ -22,7 +23,7 @@
     "advanced_options": {
       "clone_dir": "./src/github.com/Azure/azure-sdk-for-go",
       "sdk_generation_pull_request_base": "integration_branch",
-      "main_branch": "latest"
```

Comment:
Why is main_branch no longer needed?

Reply:
The main branch is now defined in the specificationRepositoryConfiguration.json file in the specification repository. That way when SDK Automation reads information about the SDK repository, it already knows which branch it should treat as the main branch. This is much better than reading the swagger_to_sdk_config.json file in the master branch to determine which branch should be treated as the main branch.